packages/theme/src/components/button.ts

```diff
@@ -73,12 +73,12 @@ function ghost(props: Props) {
 }
 
 function outline(props: Props) {
-  const { colorScheme: c } = props
+  const { colorScheme: c, theme } = props
   const borderColor = mode(`gray.200`, `whiteAlpha.300`)(props)
 
   return {
     container: {
-      border: "1px solid",
+      border: theme.borders['1px'],
```

Comment:
This is no longer necessary, I think the best solution would be to use `border : 1px` since this will automatically resolve to the theme's border value.

Reply:
That's so cool, I can't imagine this way can work haha. 🤯

So, should I need to update related to the above mentioned `border: 1px`?

Reply:
That's exactly why I left it that way. Haha

I'll close this PR for now considering you now understand what's happening.

Thank you, man.